### ISSUE TITLE
Exercise 3: Missing label

### ISSUE THREAD

**Issue by JMGaljaard** [lydiaYchen/MSGAI24#8]
In the current implementation of the homework, the exercise 3calls the test_datset.format with the wrong columsn list.

Currently, it calls with `['input_ids', 'labels']`, whereas this should be: `['input_ids', 'attention_mask', 'label',]`.

This went undetected during testing, due to a dataset with renamed columns that matched the expectation. This should be fixed.
